batavia/utils.js

```diff
@@ -791,3 +787,29 @@ batavia.make_callable = function(func) {
     fn.__python__ = true;
     return fn;
 };
+
+/*************************************************************************
+ * get the argument type as a string
+ *************************************************************************/
+ 
+ batavia.get_type = function(arg) {
```

Comment:
The location looks as good as any. The only improvement I can think of would be to use an actual class rather than a string - but that's something we can clean up later.